whitepaper/whitepaper.md

```diff
@@ -1760,4 +1761,16 @@ The blockchain migrator feature paves the way for building a solution that perfo
 
 4: H.M.N. Dilum Bandara, Xiwei Xu, and Ingo Weber. 2020. [Patterns for blockchain data migration](https://arxiv.org/abs/1906.00239). European Conf. on Pattern Languages of Programs 2020 (EuroPLoP 2020).
 
-5: Belchior,  A.  Vasconcelos,  S.  Guerreiro,  and  M.  Correia,  “A  Surveyon Blockchain Interoperability: Past, Present, and Future Trends,”arXiv,2020. [Online]. Available: http://arxiv.org/abs/2005.14282
\ No newline at end of file
+5: Belchior,  A.  Vasconcelos,  S.  Guerreiro,  and  M.  Correia,  “A  Surveyon Blockchain Interoperability: Past, Present, and Future Trends,”arXiv,2020. [Online]. Available: http://arxiv.org/abs/2005.14282
+
+<div style="page-break-after: always; visibility: hidden"><!-- \pagebreak --></div>
+
+# 10. Recommended Reference
+Please use the following BibTex entry to cite this whitepaper:
+
+@article{hyperledgercactus,
+  title={Hyperledger Cactus Whitepaper v0.1},
+  author={Montgomery, Hart and Borne-Pons, Hugo and Hamilton, Jonathan and Bowman, Mic and Somogyvari, Peter and Fujimoto, Shingo and Takeuchi, Takuma and Kuhrt, Tracy and Belchior, Rafael},
+  journal={URL: https://github. com/hyperledger/cactus/blob/main/whitepaper/whitepaper. md},
```

Comment:
typo?: "https://github. com/hyperledger/cactus/blob/main/whitepaper/whitepaper. md" should be revised to "https://github.com/hyperledger/cactus/blob/main/whitepaper/whitepaper.md" (removing the spaces)